roles/zuul/tasks/server.yml

```diff
@@ -21,3 +21,27 @@
     dest: /etc/zuul/config/bonnyci_functions.py
     owner: zuul
   notify: Reload zuul-server
+
+- name: Create zuul status lib dir
+  file:
+    path: /opt/source/zuul/etc/status/public_html/lib
+    state: directory
+
+- name: Install jquery and jquery-visibility
+  get_url:
+    url: "{{ item }}"
+    dest: /opt/source/zuul/etc/status/public_html/lib
+  with_items:
+    - http://code.jquery.com/jquery.min.js
+    - https://raw.githubusercontent.com/mathiasbynens/jquery-visibility/master/jquery-visibility.js
+    - https://raw.githubusercontent.com/prestontimmons/graphitejs/master/jquery.graphite.js
+
+- name: Create bootstrap css dir
+  file:
+    path: /opt/source/zuul/etc/status/public_html/lib/bootstrap/css
```

Comment:
I'm gonna blame bootstrap for this one, since this directory structure seems to come from the zip file we had unzipped in the previous iteration.